login-workflow/src/screens/subScreens/exportedSubScreens/ExistingAccountComplete.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { useLanguageLocale } from '@brightlayer-ui/react-auth-shared';
+import { FinishState } from '../../../components';
+import Person from '@mui/icons-material/Person';
+import { BrandedCardContainer, BrandedCardContainerProps } from '../../../components/BrandedCardContainer';
+
+/**
+ * Component that renders a screen displaying success for creating an account for
+ * a user that already exists.
+ *
+ * @category Component
+ */
+
+type ExistingAccountCompleteProps = BrandedCardContainerProps & {
+    showInCard?: boolean;
+    cardTitle?: string;
+    messageIcon?: JSX.Element;
+    messageTitle?: string;
+    messageBody?: string;
+    actions?: JSX.Element;
+
+}
```

Comment:
```suggestion
    messageIcon?: JSX.Element;
    messageTitle?: string;
    messageBody?: string;
    showInCard?: boolean;
}
```
The `cardTitle` and `actions` are never used here so we can omit them